aws/resource_aws_ses_domain_identity_verification.go

```diff
@@ -73,8 +73,16 @@ func resourceAwsSesDomainIdentityVerificationCreate(d *schema.ResourceData, meta
 
 		return nil
 	})
+	if isResourceTimeoutError(err) {
+		var att *ses.IdentityVerificationAttributes
+		att, err = getAwsSesIdentityVerificationAttributes(conn, domainName)
+
+		if aws.StringValue(att.VerificationStatus) != ses.VerificationStatusSuccess {
```

Comment:
Before we access `att` here, we need to perform a nil check on `att` otherwise this code can panic when there is an error, e.g.

```suggestion
		if att != nil && aws.StringValue(att.VerificationStatus) != ses.VerificationStatusSuccess {
```